src/torrent.cpp

```diff
@@ -5347,6 +5347,12 @@ namespace {
 #endif
 		}
 
+		if (!err)
+		{
+			if (alerts().should_post<file_prio_alert>())
+				alerts().emplace_alert<file_prio_alert>(get_handle());
+		}
+
 		if (err)
```

Comment:
`else`?

Reply:
there is an else attach to this `if` already, it wouldn't work unfortunately.